.circleci/config.yml

```diff
@@ -1229,7 +1229,7 @@ jobs:
           command: |
             apk update
             # needed for grcov and compiling tests
-            apk add --no-cache mold clang curl llvm19-dev zlib-static clang19-static lcov
```

Comment:
wasmer made their `zip` dependency optional and we don't use the feature that requires it, so we don't need this anymore (yay!)